torch/autograd/profiler.py

```diff
@@ -516,12 +509,12 @@ def _privateuse1_memory_usage(mem_record):
             )
             max_evt_id = max(max_evt_id, fe.id)
             if fe.device_type == DeviceType.CPU and not fe.is_async:
-                if self.use_device:
+                if self.use_device and self.use_device not in ["cuda", "xpu"]:
```

Comment:
privateuseone seems to set self.use_device=None, so this block may not be called. Maybe we should make privateuseone a valid str in use_device?

Reply:
+1